### DISCUSSION TITLE
NodePort connection from outside

### DISCUSSION THREAD

**Discussion by MaiTest** [OT-CONTAINER-KIT/redis-operator#117] [Q&A]
Hello
i need to connect to my cluster from outside as im building it on a baremetal so i set the service type to NodePort and set a custom port to redirect 6379 to it.

But when i connect to the cluster and set any key it can't redirect to any slot on another node and give me connection time out error!
So, how to resolve that?

**Comment by radoslaw505** [comment]
Hello,
got the same issue but i have changed `serviceType: LoadBalancer` and i can connect to cluster:
`$ redis-cli -h <url> -p 443 -a <password>` # 443 is HaProxy port
But when i do any command i got:
```
> get tommy
Error: Server closed the connection
```
